ReactNativeClient/lib/share.js

```diff
@@ -0,0 +1,13 @@
+const { NativeModules, Platform } = require('react-native');
+
+const ext = Platform.OS === 'android' ?
```

Comment:
Tried to make sure it does not break iOS but can't test myself as I do not have any iOS devices